doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -254,6 +254,11 @@ Common Application Framework (CAF)
   The :kconfig:option:`CONFIG_CAF_BLE_STATE_SECURITY_REQ` can be used to enable this feature.
   The option can be used for both Bluetooth Peripheral and Bluetooth Central.
 
+* :ref:`caf_sensor_manager`:
+
+  * Changed to not use floats to calculate if the sensor trigger is activated.
+    Also, data sent to :c:struct:`sensor_event` uses :c:struct:`sensor_value` instead of float.
```

Comment:
```suggestion
  * No longer uses floats to calculate and determine if the sensor trigger is activated.
    This is because the float uses more space.
    Also, data sent to :c:struct:`sensor_event` uses :c:struct:`sensor_value` instead of float.
```